Sources/xcodes/main.swift

```diff
@@ -119,7 +119,7 @@ let install = Command(usage: "install <version>",
         installation = .latest
     } else if flags.getBool(name: "latest-prerelease") == true {
         installation = .latestPrerelease
-    } else if let url = flags.getString(name: "url"), let path = Path(url) {
+    } else if let url = flags.getString(name: "path"), let path = Path(url) {
```

Comment:
```suggestion
    } else if let pathString = flags.getString(name: "path"), let path = Path(pathString) {
```

This might be more consistent, too.